future-apps/payroll/contracts/Payroll.sol

```diff
@@ -172,19 +176,15 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     }
 
     /**
-     * @notice Set the price feed for exchange rates to `_feed`
+     * @notice Set the price feed for exchange rates to `_feed` and the acceptable rates expiry time to `@transformTime(_time)`
      * @param _feed Address of the new price feed instance
-     */
-    function setPriceFeed(IFeed _feed) external authP(MODIFY_PRICE_FEED_ROLE, arr(_feed, feed)) {
-        _setPriceFeed(_feed);
-    }
-
-    /**
-     * @notice Set the acceptable expiry time for the price feed's exchange rates to `@transformTime(_time)`
-     * @dev Exchange rates older than the given value won't be accepted for payments and will cause payouts to revert
      * @param _time The expiration time in seconds for exchange rates
      */
-    function setRateExpiryTime(uint64 _time) external authP(MODIFY_RATE_EXPIRY_ROLE, arr(uint256(_time), uint256(rateExpiryTime))) {
+    function setPriceFeedSettings(IFeed _feed, uint64 _time)
```

Comment:
I unified `setPriceFeed` and `setRateExpiryTime` functions to reduce the public interface of the payroll app, we were hitting the max size limit. Currently the deployed bytecode is ~24.1k